libs/application-generic/src/services/cache/interceptors/cached-entity.interceptor.ts

```diff
@@ -89,7 +89,9 @@ export function CachedEntity<T_Output = any>({
             `Failed to acquire lock for key: ${cacheKey} in "method: ${methodName}"`,
             LOG_CONTEXT,
           );
-          throw new Error(`Failed to acquire lock for key: ${cacheKey}`);
+          throw new ConflictException(
+            `Failed to acquire cache lock. Please try again later.`,
+          );
```

Comment:
It was a mistake to include `cacheKey` in the lock as it can expose server implementation details. We just return a simple error message. Note, it's still not ideal that cache issues should throw a client error.